src/controllers/popularResController.ts

```diff
@@ -0,0 +1,17 @@
+import express from "express";
+import { RestaurantModel } from "../models/restaurant";
+
+export const getPopularRestaurants = async (
+  req: express.Request,
+  res: express.Response
+) => {
+  try {
+    const popularRestaurants = await RestaurantModel.find({
+      popularRestaurant: true,
+    });
```

Comment:
find is giving only the first element, u'll need to use the filter on the array

Reply:
fixed